packages/core/src/interaction-output.ts

```diff
@@ -33,7 +33,7 @@ const ajv = new Ajv({ strict: false });
 
 export class InteractionOutput implements WoT.InteractionOutput {
     private content: Content;
-    private parsedValue: unknown;
+    #value: unknown;
```

Comment:
Mhh, my feeling is that using `#` here inconsistent with the other variables. Either we change all or none I would say. Changing the name from `parsedValue`to `value` is fine by me.

Reply:
good point I was not sure how to align all of them. I'll add a refactor commit tomorrow. Yeah, changing the name to `value` is for having a nice 1-1 mapping with the spec.

Reply:
In the current state of the class I would favor using `value` as the the variable name without `#`

Reply:
We can't because we have already a method called `value`.

Reply:
Mhh, then we need to pick another name _or_ would you go with `#value` (I wouldn't 🤷‍♂️)

Reply:
Given that the specs says that the cached result is an internal slot called value, the 1-1 mapping with JS is having it to be `#value`. I like the coherence of the algorithm that we wrote, but I found calling it `#value` a bit odd too. It's a tradeoff that we have to decide. Maybe @JKRhb what's your take here?

Reply:
Yup, maybe @JKRhb has a good advice.

BTW, qhat about changing the other variables to # notation too. Doing so makes sense to me (even though a bit unrelated to this PR/change).

Reply:
Maybe we could call it something like `internalValue`? This would correspond with the "internal slots" phrasing. However, I think I would also prefer the `#value` approach – couldn't we simply rename the other private variables as well (e.g., changing `content` to `#content`)?

Reply:
> couldn't we simply rename the other private variables as well (e.g., changing `content` to `#content`)?

That's what I meant as well

Reply:
> > couldn't we simply rename the other private variables as well (e.g., changing `content` to `#content`)?
> 
> That's what I meant as well

Ah, sorry, I should have refreshed the page before posting my comment 😅🙈

Reply:
Should be solved now in c10cdfc